client/fingerprint/env_aws_cpu.go

```diff
@@ -245,13 +262,15 @@ var instanceTypeCPU = map[string]CPU{
 	"m6gd.medium":   newCPU(1, 2.5),
 	"m6gd.metal":    newCPU(64, 2.5),
 	"m6gd.xlarge":   newCPU(4, 2.5),
+	"mac1.metal":    newCPU(12, 3.2),
```

Comment:
I can't wait to start getting some wild Mac-on-AWS bug reports 😀

Reply:
:sweat_smile: